dist/angular-chartist.js

```diff
@@ -22,17 +22,15 @@
         this.responsiveOptions = $scope.responsiveOptions() || null;
     }
 
+    AngularChartistCtrl.$inject = ['$scope'];
```

Comment:
Actually that is great! I have been meaning to switch to this style over the dep array.